improver/cli/apply_reliability_calibration.py

```diff
@@ -37,7 +37,7 @@
 @cli.clizefy
 @cli.with_output
 def process(
-    forecast: cli.inputcube, reliability_table: cli.inputcube = None,
+    forecast: cli.inputcube, reliability_table: cli.inputcubelist = None,
```

Comment:
The doc-string below should be modified to indicate that reliability_table can be a cube or cubelist.